lib/protocol/context.js

```diff
@@ -25,6 +25,6 @@ export default function context (id) {
         data.name = id
     }
 
-    deprecate('context')
+    deprecate('context', this.options)
```

Comment:
Can we remove the deprecate warning here? This is an Appium command and won't be removed anytime soon.